org.lflang/src/lib/cpp/lfutil.hh

```diff
@@ -27,6 +27,8 @@
 #include <reactor-cpp/reactor-cpp.hh>
 #include <reactor-cpp/logging.hh>
 
+#include <list>
```

Comment:
I guess `std::list<T>`  is already in use but I can tell u from experience `std::list<T>`  ist a big performance killer

Reply:
It is just used during instantiation, so it shouldn't be too bad